.config/webpack/utils.ts

```diff
@@ -1,42 +1,59 @@
 import fs from 'fs';
+import process from 'process';
+import os from 'os';
 import path from 'path';
 import util from 'util';
-import glob from 'glob';
+import { glob } from 'glob';
 import { SOURCE_DIR } from './constants';
 
-const globAsync = util.promisify(glob);
+
+export function isWSL() {
```

Comment:
Could this function go in a shared lib? Seen this in a couple of places now, it's probably quite useful

Reply:
This is autogenerated based on `npx @grafana/create-plugin@latest update` so I guess technically already in a shared lib 😄